manifests/params.pp

```diff
@@ -99,6 +99,8 @@
   $max_keep_alive = 10000
   $num_workers = min($::processorcount, 8)
 
+  $puppet_wsgi_processes = 3
```

Comment:
I noticed that after I made the comment. That's fine then, if needed it can be tuned later like num_workers above.